tools/topology/m4/utils.m4

```diff
@@ -114,5 +114,43 @@ define(`VIRTUAL_WIDGET',
 `       no_pm "true"'
 `}', `fatal_error(`Invalid parameters ($#) to VIRTUAL_WIDGET')')')
 
+ifdef(`SOFABIPATH', `define(`ABI_PATH', SOFABIPATH)', `define(`ABI_PATH', ../../../src/include/kernel/abi.h)')
```

Comment:
The path is given to command line system call egrep, so I don't know how to forward it from m4 include... anyway I changed the definition. As the SOF_ABI_VERSION is composed by cmake, at least CI/Docker will find it (when building tools). You can give it from command line now as -DSOF_ABI_FILE= if you want to compile individual demux topology file by hand. Otherwise the macro is empty.